.gitignore

```diff
@@ -169,3 +169,5 @@ dmypy.json
 
 # End of https://www.gitignore.io/api/macos,python,visualstudiocode
 env.list
+prod.env
+debug.env
```

Comment:
I just made this `*.env` in mine in case we wanted to do any other ones

Reply:
yeah i saw that. i'll rebase on top of your changes after yours gets merged and remove this.